src/pages/workspaces/workspace/Workflows.js

```diff
@@ -237,9 +289,17 @@ const FindWorkflowModal = ajaxCaller(class FindWorkflowModal extends Component {
     const { namespace, name, snapshotId } = selectedWorkflow
 
     this.setState({ selectedWorkflow })
+
     try {
       const selectedWorkflowDetails = await Methods.method(namespace, name, snapshotId).get()
-      this.setState({ selectedWorkflowDetails })
+
+      const methodAjax = Ajax().Methods.method(selectedWorkflow.namespace, selectedWorkflow.name, selectedWorkflow.snapshotId)
```

Comment:
nit: `selectedWorkflow` was already destructured above, so we can use the extracted variables directly here.